Crypt/Mechanisms/CryptMechanism.swift

```diff
@@ -152,7 +152,7 @@ class CryptMechanism: NSObject {
   var needsEncryption: Bool {
     set {
       os_log("needsEncryption set to %@", log: CryptMechanism.log, type: .default, newValue as CVarArg)
-      let data = NSKeyedArchiver.archivedData(withRootObject: NSNumber.init(value: newValue))
+      let data = try! NSKeyedArchiver.archivedData(withRootObject: NSNumber.init(value: newValue), requiringSecureCoding: false)
```

Comment:
@sheagcraig @grahamgilbert @og-joel I'm gonna admit this was pure poke and hope. Please advise.

Reply:
set {
      os_log("needsEncryption set to %@", log: CryptMechanism.log, type: .default, newValue as CVarArg)
      guard let data = try? NSKeyedArchiver.archivedData(withRootObject: NSNumber.init(value: newValue), requiringSecureCoding: false) else { return }
      _ = setHintData(key: needsEncryptionHintKey, data: data as NSData)
    }

Reply:
Updated thanks!